### PR TITLE
[zh-cn] sync contribute/* container-runtimes

### PR DESCRIPTION
**Pull request by my-git9**
```
# Changes to be committed:
#       modified:   content/zh-cn/docs/contribute/_index.md
#       modified:   content/zh-cn/docs/contribute/new-content/blogs-case-studies.md
#       modified:   content/zh-cn/docs/contribute/style/page-content-types.md
#       modified:   content/zh-cn/docs/contribute/style/style-guide.md
#       modified:   content/zh-cn/docs/contribute/style/write-new-topic.md
#       modified:   content/zh-cn/docs/contribute/suggesting-improvements.md
#       modified:   content/zh-cn/docs/setup/production-environment/container-runtimes.md

```

### CODE REVIEW

File: content/zh-cn/docs/contribute/_index.md [hunk 1]
```diff
@@ -240,6 +243,27 @@ Figure 2. Preparation for your first contribution.
...
+preferably in the `#sig-docs` channel. There is also the
+[New Contributors Meet and Greet call](https://www.kubernetes.dev/resources/calendar/)
+that happens on the first Tuesday of every month. You can interact with
+the New Contributor Ambassadors and get your queries resolved here.
+-->
+## 贡献时获取帮助
+
+做出第一个贡献可能会让人感觉比较困难。
+[新贡献者大使](https://github.com/kubernetes/website#new-contributor-ambassadors)
+将引导您完成最初的一些贡献。你可以在 [Kubernetes Slack](https://slack.k8s.io/)
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/_index.md:262]:
```suggestion
将协助你完成最初的一些贡献。你可以在 [Kubernetes Slack](https://slack.k8s.io/)
```

File: content/zh-cn/docs/contribute/suggesting-improvements.md [hunk 2]
```diff
@@ -4,21 +4,29 @@ content_type: concept
 weight: 10
 card:
   name: contribute
-  weight: 20
+  weight: 15
+  anchors:
+    anchor: "#opening-an-issue"
+    title: 提出内容改进建议
```
Comment (tengqm) [inline content/zh-cn/docs/contribute/suggesting-improvements.md:10]:
```suggestion
    - anchor: "#opening-an-issue"
      title: 提出内容改进建议
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**